content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -30,12 +30,37 @@ Refer to the [traffic management](/docs/tasks/traffic-management/) documentation
 
 ## Security
 
-Without a waypoint installed, you can only use [Layer 4 security policies](/docs/ambient/usage/l4-policy/).  By adding a waypoint, you gain access to the following policies:
+Without a waypoint installed, you can only use [Layer 4 security policies](/docs/ambient/usage/l4-policy/). By adding a waypoint, you gain access to the following policies:
 
 |  Name  | Feature Status | Attachment |
 | --- | --- | --- |
-| `AuthorizationPolicy` (including L7 features) | Beta | `targetRefs` |
-| `RequestAuthentication` | Beta | `targetRefs` |
+| [`AuthorizationPolicy`](/docs/reference/config/security/authorization-policy/) (including L7 features) | Beta | `targetRefs` |
+| [`RequestAuthentication`](/docs/reference/config/security/request_authentication/) | Beta | `targetRefs` |
+
+### Considerations for authorization policies
+
+Authorization policies whose [conditions](/docs/reference/config/security/conditions/) only target Layer 4 (TCP) will either be [enforced by the ztunnel or the waypoint](/docs/ambient/usage/l4-policy#considerations) depending on where the policy is targeted for attachment.
+
+In a scenario where a policy contains conditions that match L7 attributes (for example, HTTP verbs), a waypoint proxy is **required**. It is important to understand that ztunnel cannot meaningfully enforce any policy that requires L7 parsing. If an authorization policy has been configured that requires any traffic processing beyond L4, and if no waypoint proxies are configured for the destination of the traffic, then **the ztunnel proxy will DENY all traffic** as a defensive move.
```

Comment:
To be clear: if the policy is attached via something that causes it to be enforced by the ztunnel, /and/ it contains L7 conditions, it will become a `DENY`?

What happens if a policy exists that would bind it to a waypoint (`targetRef` to its `Service`, say) but that waypoint doesn't exist?

Reply:
If there is no waypoint traffic doesn't go to a waypoint to enforce anything. The policy would be ignored basically as there is nothing to enforce it.

Same goes for if you tell a service to use a non-existent waypoint IIRC.